packages/esm-billing-app/src/billable-services/billable-service.resource.tsx

```diff
@@ -38,6 +38,16 @@ export const createBillableService = (payload: any) => {
     },
   });
 };
+export const deleteBillableService = (payload: any) => {
+  const url = `/ws/rest/v1/cashier/api/deletebillable-service`;
```

Comment:
```suggestion
  const url = `${restBaseUrl}/cashier/api/deletebillable-service`;
```
this is a soft delete to have the complete delete we append the param `?purge=true`

Reply:
@donaldkibet the endpoint does not  exist am waiting for @makombe  to have this then i will finalize